CONTRIBUTING.md

```diff
@@ -66,6 +66,12 @@ Before you can start contributing to the CyberArk Secrets Provider for Kubernete
 
 From here your pull request is reviewed. Once you have implemented all reviewer feedback, your code is merged into the project. Congratulations, you're a contributor!
 
+### Development
+
+To spin up a development environment, run `./bin/start --dev` and concatenate the environment (--dap / --oss) and 
+platform flag (--gke / --oc311 / oc310) of your choice. By default, the script deploys OSS to GKE, so no extra 
+flags would be needed.
```

Comment:
can you add an example? something like "For example, to spin up a development environment with a DAP cluster that runs on OCP 3.10 run `./bin/start --dev --dap --oc310`"